index.js

```diff
@@ -317,16 +327,23 @@ class K8sVMExecutor extends Executor {
                         `${JSON.stringify(resp.body.status, null, 2)}`);
                 }
 
+                const updateConfig = {
+                    apiUri: this.ecosystem.api,
+                    buildId,
+                    token
+                };
+
+                if (resp.body.spec && resp.body.spec.nodeName) {
+                    updateConfig.stats = {
+                        hostname: resp.body.spec.nodeName
```

Comment:
if build's stats has `queueEnterTime` in it, this will override that.

Reply:
It will handle the merging logic inside the API
this won't merge or override

Reply:
👍